solr/core/src/java/org/apache/solr/cloud/SizeLimitedDistributedMap.java

```diff
@@ -91,9 +91,13 @@ protected boolean lessThan(Long a, Long b) {
       for (String child : children) {
         Long id = childToModificationZxid.get(child);
         if (id != null && id <= topElementMzxId) {
-          zookeeper.delete(dir + "/" + child, -1, true);
-          if (onOverflowObserver != null)
-            onOverflowObserver.onChildDelete(child.substring(PREFIX.length()));
+          try {
+            zookeeper.delete(dir + "/" + child, -1, true);
+            if (onOverflowObserver != null)
+              onOverflowObserver.onChildDelete(child.substring(PREFIX.length()));
+          } catch (KeeperException.NoNodeException e) {
```

Comment:
nit: naming the exception `ignore` or `ignored` I think makes IDEs not care that the exception isn't used. It doesn't change the behavior at all just clear we meant to ignore the exception.

Reply:
👍🏼  changed